### ISSUE TITLE
Xonsh 0.8.11 dies while starting

### ISSUE THREAD

**Issue by rapha8l** [xonsh/xonsh#3037]
Requirement already up-to-date: xonsh in /usr/lib/python3.6/site-packages (0.8.11)

### Traceback (if applicable)

<details>

```

Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 24004, in main
    args = premain(argv)
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 23955, in premain
    env = start_services(shell_kwargs, args)
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 23890, in start_services
    cacheall=shell_kwargs.get("cacheall", False),
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 23021, in __init__
    load_builtins(execer=self, ctx=xonsh_ctx)
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 22592, in load_builtins
    builtins.__xonsh__.load(execer=execer, ctx=ctx)
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 22668, in load
    self.env = Env(default_env())
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 19110, in __init__
    self[key] = val
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 19272, in __setitem__
    val = ensurer.convert(val)
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 18218, in convert
    return cls.fromstring(x)
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 18181, in fromstring
    esc, "default", reversed_style=reversed_default
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 7022, in ansi_color_escape_code_to_name
    short_to_ints(short), background=(fore_back == "BACK"), prefix=pre
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 1268, in short_to_ints
    return rgb_to_ints(short2rgb(short))
  File "/usr/lib/python3.6/site-packages/xonsh/__amalgam__.py", line 1199, in short2rgb
    return SHORT_TO_RGB[short]
KeyError: '5'
Xonsh encountered an issue during launch
Failback to /bin/sh

```
</details>

Thanks

**Comment by scopatz** [comment]
Thanks for reporting! I am working on a fix now.

**Comment by scopatz** [comment]
There is a potential fix in #3045, if you'd like to check that out

**Comment by rapha8l** [comment]
not sure, it's significant, but I just cloned and used the docker command and it works (I already had deleted all dotfiles between last crashes, so it should be the same, maybe)
